compiler/rustc_resolve/src/macros.rs

```diff
@@ -689,6 +691,32 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         res.map(|res| (self.get_macro(res).map(|macro_data| macro_data.ext.clone()), res))
     }
 
+    fn report_invalid_crate_level_attr(&mut self, attrs: &[Attribute], name: Symbol) -> bool {
+        for attr in attrs.iter().filter(|attr| attr.style == AttrStyle::Inner) {
```

Comment:
```suggestion
        for attr in attrs {
```
Redundant condition, crate attributes are inner by definition.